### ISSUE TITLE
Unable to run BraintreeASPExample 

### ISSUE THREAD

**Issue by BrianHo888** [braintree/braintree_aspnet_example#27]
Unable to run BraintreeASPExample by following the ReadMe file.   
![2018-10-30_10-12-58](https://user-images.githubusercontent.com/31257988/47736695-79817880-dc2c-11e8-9ff2-94e7f47ff2b1.jpg)

* Environment: <SandBox>
* Language, language version, and OS:

**Comment by crookedneighbor** [comment]
This is likely related to #11 where we have not updated the ASP.NET version to the latest.

**Comment by m-henderson** [comment]
@crookedneighbor true. I played around with it and got it working though. @BrianHo888 try opening it up in vs2012 if you have it.

**Comment by hollabaq86** [comment]
for internal tracking, ticket 1882
